packages/react/src/VideoPlayer/components/controls/Controls.tsx

```diff
@@ -0,0 +1,304 @@
+              strokeLinecap="round"
+            />
+          </svg>
+        )}
+      </IconControl>
+    ),
+    [volume, previousVolume],
+  )
+
+  /* 5. Volume                                                  */
+  useEffect(() => {
+    if (videoRef.current) videoRef.current.volume = volume
+  }, [videoRef, volume])
+
+  const renderVolume = useMemo(
+    () => (
+      <Range
+        type="range"
+        min="0"
+        max={1}
+        step={0.001}
+        onInput={e => {
+          setVolume(e.currentTarget.valueAsNumber)
+        }}
+        className={styles.VideoPlayer__volumeBar}
+        value={volume}
+        a11yStep={0.1}
+        id="VideoPlayer__Volume"
+      />
+    ),
+    [volume],
+  )
+
+  const renderFullscreenButton = useMemo(() => {
+    const handleFullScreen = () => {
+      if (videoWrapperRef.current) {
+        if (!fullScreen) {
+          videoWrapperRef.current.requestFullscreen()
+          setFullScreen(true)
+        } else {
+          document.exitFullscreen()
+          setFullScreen(false)
+        }
+      }
+    }
+
+    return (
+      <IconControl onClick={handleFullScreen} tooltip={!fullScreen ? 'Full Screen' : 'Exit Full Screen'}>
+        {!fullScreen ? (
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
+            <path
+              fillRule="evenodd"
+              clipRule="evenodd"
+              d="M3.13888 2.83333C2.97013 2.83333 2.83333 2.97013 2.83333 3.13888V8.33331C2.83333 8.83957 2.42292 9.31108 1.91666 9.31108C1.4104 9.31108 1 8.83957 1 8.33331V3.13888C1 1.95761 1.95761 1 3.13888 1H8.33331C8.83957 1 9.31108 1.47152 9.31108 1.97777C9.31108 2.48403 8.83957 2.83333 8.33331 2.83333H3.13888ZM14.6888 1.97777C14.6888 1.47152 15.1604 1 15.6666 1H20.861C22.0423 1 22.9999 1.95761 22.9999 3.13888V8.33331C22.9999 8.83957 22.5895 9.31108 22.0833 9.31108C21.577 9.31108 21.1666 8.83957 21.1666 8.33331V3.13888C21.1666 2.97013 21.0298 2.83333 20.861 2.83333H15.6666C15.1604 2.83333 14.6888 2.48403 14.6888 1.97777ZM1.91666 14.6888C2.42292 14.6888 2.83333 15.1604 2.83333 15.6666V20.861C2.83333 21.0298 2.97013 21.1666 3.13888 21.1666H8.33331C8.83957 21.1666 9.31108 21.5159 9.31108 22.0222C9.31108 22.5284 8.83957 22.9999 8.33331 22.9999H3.13888C1.95761 22.9999 1 22.0423 1 20.861V15.6666C1 15.1604 1.4104 14.6888 1.91666 14.6888ZM22.0833 14.6888C22.5895 14.6888 22.9999 15.1604 22.9999 15.6666V20.861C22.9999 22.0423 22.0423 22.9999 20.861 22.9999H15.6666C15.1604 22.9999 14.6888 22.5284 14.6888 22.0222C14.6888 21.5159 15.1604 21.1666 15.6666 21.1666H20.861C21.0298 21.1666 21.1666 21.0298 21.1666 20.861V15.6666C21.1666 15.1604 21.577 14.6888 22.0833 14.6888Z"
+              fill="currentColor"
+              stroke="#fff"
+              strokeWidth="1.5"
+              strokeLinecap="round"
+            />
+          </svg>
+        ) : (
+          <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none">
```

Comment:
Same comment about using octicons for these, as we seem to have equivalents: https://primer.style/design/foundations/icons/screen-full-16

Reply:
This has been updated in this SHA: https://github.com/primer/brand/pull/373/commits/3476819281a2ff27b50715bf57261befc6f1d0f5